sonar-bukkit/src/main/java/xyz/jonesdev/sonar/bukkit/listener/BukkitJoinListener.java

```diff
@@ -24,69 +24,64 @@
 import org.bukkit.event.player.PlayerJoinEvent;
 import org.jetbrains.annotations.NotNull;
 import xyz.jonesdev.sonar.api.Sonar;
-import xyz.jonesdev.sonar.api.fallback.FallbackPipelines;
 import xyz.jonesdev.sonar.bukkit.fallback.FallbackBukkitInjector;
 
 import java.lang.invoke.MethodHandle;
 import java.lang.invoke.MethodHandles;
 import java.lang.reflect.Field;
 import java.lang.reflect.Method;
-import java.lang.reflect.Modifier;
 
-// Credits: https://github.com/ViaVersion/ViaVersion/blob/master/bukkit/src/main/java/com/viaversion/viaversion/bukkit/listeners/JoinListener.java
+import static xyz.jonesdev.sonar.api.fallback.FallbackPipelines.FALLBACK_BANDWIDTH;
+
+// Taken from
+// https://github.com/ViaVersion/ViaVersion/blob/master/bukkit/src/main/java/com/viaversion/viaversion/bukkit/listeners/JoinListener.java
 public final class BukkitJoinListener implements Listener {
   // ServerGamePacketListenerImpl aka. PlayerConnection
```

Comment:
Delete this line or move to `packetListenerField`??

Reply:
Well, we took more than the field stuff, so I guess it's fine to attribute the entire class to VV.

Reply:
i mean `ServerGamePacketListenerImpl aka. PlayerConnection`

Reply:
> i mean `ServerGamePacketListenerImpl aka. PlayerConnection`

Oh, yeah, you're right.

Reply:
https://github.com/jonesdevelopment/sonar/pull/358/commits/d32cdda6e861c329e8ac4b14fe93e843dbf393c3